packages/requisitions/src/ResponseRequisition/api/api.ts

```diff
@@ -1,4 +1,5 @@
 import {
+  RequisitionNodeStatus,
```

Comment:
Just the same as request. Changed `ResponseQuery` object to use a closure for the `sdk`/`storeId` when initially created rather than passing to individual functions